### ISSUE TITLE
Does not fix issue.

### ISSUE THREAD

**Issue by DennisCullen** [dotnet/docs#27057]
[Enter feedback here]
This does not fix the error.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2f6f22d0-6953-a07a-032d-749638f0cedc
* Version Independent ID: 8e72af9b-36c4-4775-466f-5860efcfe265
* Content: [No accessible 'Main' method with an appropriate signature was found in '&lt;name&gt;' - Visual Basic](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/error-messages/no-accessible-main-method-with-an-appropriate-signature-was-found-in-name)
* Content Source: [docs/visual-basic/language-reference/error-messages/no-accessible-main-method-with-an-appropriate-signature-was-found-in-name.md](https://github.com/dotnet/docs/blob/main/docs/visual-basic/language-reference/error-messages/no-accessible-main-method-with-an-appropriate-signature-was-found-in-name.md)
* Product: **dotnet-visualbasic**
* Technology: **vb-diagnostics**
* GitHub Login: @KathleenDollard
* Microsoft Alias: **kdollard**

**Comment by Youssef1313** [comment]
Can you upload a sample code/project where you followed the instructions and the error still exists?

**Comment by DennisCullen** [comment]
No.  I'm closing this because I was confused.  I was going through a sample for the new VS 2022.  There are two projects in the solution, one is a NET core class, and the other is a NET console app.  I keep misreading which of the two projects was having the issue.  I found the samples file naming a bit unclear. Anyway, sometime after submitting my comment to this thread I eventually figured it out.
